fetch-crypto-assets.js

```diff
@@ -0,0 +1,65 @@
+const file = "./data.json"
+
+async function getJSON(jsonFile) {
+    try {
+        const response = await fetch(jsonFile)
+        const data = await response.json()
+        return data.cryptoAssets
+    } catch (e) {
+        throw new Error("Retrieving JSON failed", e)
+    }
+}
+
+function buildURL(param) {
+    return `https://api.coinbase.com/v2/prices/${param}/spot`
+}
+
+async function getURLS() {
+    try {
+        const params = await getJSON(file)
```

Comment:
does `const params = file` not work? 🤔

Reply:
After our conversation, cannot use `module.exports` in a non-Node environment. This is just pure vanilla JS.